sdk/python/feast/cli/cli.py

```diff
@@ -91,7 +93,8 @@ def format_options(self, ctx: click.Context, formatter: click.HelpFormatter):
 @click.option(
     "--feature-store-yaml",
     "-f",
-    help="Override the directory where the CLI should look for the feature_store.yaml file.",
+    envvar=FEAST_FS_YAML_FILE_PATH_ENV_NAME,

```

Comment:
```suggestion
    envvar=FS_YAML_FILE_PATH_ENV_VAR,

```
FEAST is implcit here.

Reply:
I simply removed it from latest commit.

Reply:
Interesting, How setting up that as EnvVar would be recognized then ?

Reply:
https://github.com/feast-dev/feast/blob/master/sdk/python/feast/utils.py#L96

Reply:
oh interesting, i guess we'll follow the FEAST prefix. feels bad 🤷